docs/attributes.md

```diff
@@ -36,8 +36,10 @@ Configurable (with defaults)
 * `node['splunk']['config']['secrets']` - Coordinate String (see [data bags documentation][data_bags]), pointing to a key within a Chef Vault encrypted data bag item used to configure the splunk.secret file. The value must be a string. (`nil` - secrets not managed by chef).  Note: this is currently not supported on windows.
 * `node['splunk']['config']['licenses']` - Data bag item that the license server recipe uses as the source of truth for the license data.
 * `node['splunk']['config']['license-pool']` - Data bag item used to configure license pools.
-* `node['splunk']['config']['ui_prefs']` - Hash of stanzas used to configure [ui-prefs.conf][] on the search head in a clustered configuration or a standalone instance.
+* `node['splunk']['config']['ui_prefs']` - Hash of stanzas used to configure [ui-prefs.conf][] on the search head (SH) in a clustered configuration or a standalone instance.
 * `node['splunk']['config']['assumed_index']` - Name of the index to which data is forwarded to by default, when the index is not configured for the input.(`main`)
+* `node['splunk']['config']['sh_cluster']` - Set this attribute to true to enable search head clustering. (`false`)
+* `node['splunk']['config']['sh_member']` - Specify the action to be performed on the SH member. Set this attribute to `add` to add a new SH member to the SH cluster. If set to `remove`, it removes the member from the SH cluster. Add members to the SH cluster using this attribute only if there is at least one member in the Cluster. Also note that you can only add a clean SH instance to the cluster. (`nil`)
```

Comment:
thoughts on renaming this config to sh_member_action? That would make it a bit more descriptive.